### ISSUE TITLE
Design project status badge

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#7492]
See https://github.com/cityofaustin/atd-data-tech/issues/7036. 

We want a badge/label showing project status, similar to Github: 
![Screen Shot 2021-10-14 at 3 58 09 PM](https://user-images.githubusercontent.com/1463708/137394731-098d5ffb-c285-4227-8ede-0797ad72eaab.png)
- I'm thinking it would go in the top bar of a project so that it's visible regardless what tab one is on. 
- Icon and/or color symbology could match Moped Viewer

For colors, maybe:
Grey (or red if we have to :P )
- Cancelled
- On Hold

Yellow
- Potential

Green
- Planned
- Preliminary engineering
- Scoping
- Preliminary design
- Design
- Pre-construction
- Bid/Award/Execution
- Construction-ready
- Construction
- Post-construction

Blue
- Complete

**Comment by amenity** [comment]
@patrickm02L - I'd love it if you have time to do this, but happy to take a shot if you don't.

**Comment by amenity** [comment]
@patrickm02L - I really like where this is going. Wondering about the icons... Are you proposing a different one for each status? I didn't see any others on the XD and wonder if we might should just go with ▶️ for all the greens.

**Comment by patrickm02L** [comment]
Updated based on feedback. We've decided to go with the Solid badges. 

font-weight: medium (500)
font-size: 16pt (21px)
font-family: Roboto


F34335 - Red
E0E0E0 - Grey
FFCC66 - Yellow
2DA44E - Green
1276D1 - Blue

Icons can be found [here](https://fonts.google.com/icons?selected=Material+Icons+Outlined:assignment_returned&icon.query=circle).

![Badges – 2.png](https://images.zenhubusercontent.com/5d68810b05d7792664be403f/89335fb9-a08e-4252-8de2-637e60e0baf7)
